target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -304,6 +305,55 @@ function __rspamd__setup_check_authenticated() {
   fi
 }
 
+# If `SPAM_SUBJECT` is not empty, we create a Sieve script that alters the `Subject`
+# header, in order to prepend a user-defined string.
+function __rspamd__setup_spam_subject() {
```

Comment:
This is no longer rspamd specific, the method doesn't belong in `rspamd.sh` anymore, and it should not be reliant on it's related helper for logging?

Reply:
Given the context with Amavis+SA still being supported, this could be left as-is, and addressed in a follow-up PR separately.

I think it would be better to transition Amavis+SA to using the same approach as the plans for DMS is to switch to rspamd by default and potentially remove Amavis+SA at a later date, it'd help prep for that switch if we have user reports about the subtle functionality changes involved (_only our Dovecot service provides the functionality, no more `_SCORE_` tag support_).

Reply:
This will be solved with the next commits.